pages/common/git-daemon.md

```diff
@@ -0,0 +1,16 @@
+# git daemon
+
+> A really simple server for Git repositories.
+> More information: <https://git-scm.com/docs/git-daemon>.
+
+- Git daemon with a whitelisted set of directories:
+
+`git daemon --export-all {{path/to/directory1}} {{path/to/directory2}}`
+
+- Git daemon with a base directory and allow pulling from all directories that look like Git repos:
+
+`git daemon --base-path={{path/to/directory}} --export-all --reuseaddr`
+
+- Git daemon with a base directory with allowed push access and verbosity:
+
+`git daemon --base-path={{path/to/directory}} --export-all --enable=receive-pack --reuseaddr --informative-errors --verbose`
```

Comment:
This seems like a lot to pack into a single example. There's plenty of space left on this page, so maybe we could split it up into multiple examples?

Reply:
git-daemon doesn't have much options (it's one of those things that nobody will ever use), so I think it's fine as it is

Reply:
Maybe you should move `--informative-errors --verbose` to another example about verbosity.  This example looks _unnecessarily_ ugly.

Reply:
Nevermind; I see you've documented the other flags previously.

Reply:
If a command doesn't have many options, that's a great reason to split them up into multiple examples. Remember, in tldr pages the goal is clarity and readability to assist readers in understanding how to use the command in question with the lowest possible cognitive load.